StripeIdentity/StripeIdentity/Source/API Bindings/Models/VerificationPage/VerificationPageStaticContentDocumentCaptureMBSettings.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  VerificationPageStaticContentDocumentCaptureMBSettings.swift
+//  StripeIdentity
+//
+//  Created by Chen Cen on 1/24/24.
+//
+
+import Foundation
+@_spi(STP) import StripeCore
+
+extension StripeAPI {
+    /// Policy to configure capture strategy used to select the best frame.
+    enum MBCaptureStrategyType: String, Codable, Equatable, CaseIterable {
+        case singleFrame = "single_frame"
+        case optimizeForQuality = "optimize_for_quality"
+        case optimizeForSpeed = "optimize_for_speed"
+        case defaultValue = "default"
+    }
+
+    /// Policy used to detect tilted documents.
+    enum MBTiltPolicyType: String, Codable, Equatable, CaseIterable {
+        case disabled = "disabled"
+        case normal = "normal"
+        case relaxed = "relaxed"
+        case strict = "strict"
+    }
+
+    /// Policy used to discard frames with blurred documents.
+    enum MBBlurPolicyType: String, Codable, Equatable, CaseIterable {
+        case disabled = "disabled"
+        case normal = "normal"
+        case relaxed = "relaxed"
+        case strict = "strict"
+    }
+
+    /// Policy used to discard frames with glare detected on the document.
+    enum MBGlarePolicyType: String, Codable, Equatable, CaseIterable {
+        case disabled = "disabled"
+        case normal = "normal"
+        case relaxed = "relaxed"
+        case strict = "strict"
```

Comment:
`= "..."` shouldn't be required in the case when the case es exactly the string.